test/Project.toml

```diff
@@ -16,4 +18,4 @@ Interpolations = "0.13, 0.14, 0.15"
 LegendDataTypes = "0.1"
 Measurements = "2"
 StatsBase = "0.33, 0.34"
-Unitful = "1"
\ No newline at end of file
+Unitful = "1"
```

Comment:
I would recommend using the same `[compat]` entries as in the main `Project.toml`:
```suggestion
TypedTables = "1.2"
Unitful = "1.6"
```
and add `LegendDataManagement` in all versions for which `ljl_propfunc` was defined and working: `0.3, 0.4, 0.5`.

Reply:
![image](https://github.com/user-attachments/assets/c8f5874b-b98b-4184-a96e-4c9a8d179830)
Tests on downgrade are failing because it's pulling `LegendDataManagement@0.2.3` where it was not yet defined.